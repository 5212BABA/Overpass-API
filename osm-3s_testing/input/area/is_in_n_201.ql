node(20126);is_in;out geom;
