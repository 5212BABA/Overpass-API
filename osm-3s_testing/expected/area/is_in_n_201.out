<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <way id="201">
    <bounds minlat="51.2500001" minlon="7.1500001" maxlat="51.2510001" maxlon="7.1600001"/>
    <nd ref="20101" lat="51.2500001" lon="7.1500001"/>
    <nd ref="20102" lat="51.2500001" lon="7.1600001"/>
    <nd ref="20103" lat="51.2510001" lon="7.1600001"/>
    <nd ref="20104" lat="51.2510001" lon="7.1500001"/>
    <nd ref="20101" lat="51.2500001" lon="7.1500001"/>
  </way>

</osm>
